blocks/global-phrases-block/intl.json

```diff
@@ -130,5 +130,16 @@
 		"no": "(Åpnes i et nytt vindu)",
 		"pt": "(Abre numa nova janela)",
 		"sv": "(Öppnas i nytt fönster)"
+	},
+	"global.sponsored-content": {
```

Comment:
did you merge in the update from the base branch?

Reply:
merged